Specs/DataSources/DataSourceDisplaySpec.js

```diff
@@ -280,6 +281,22 @@ defineSuite([
         expect(source2Visualizer.updatesCalled).toEqual(1);
     });
 
+    it('ready is true when datasources are ready', function() {
```

Comment:
Change this test to check that both true and false get returned depending on the values returned by update (i.e. have one data source return true and then have it return false and check ready after both times).  You can spy on MockVisulizer and have it return whatever you want.